identifiers/country-do/README.md

```diff
@@ -0,0 +1,10 @@
+# OCD Division Identifier 
+
+This folder list the following administrative divisions of Dominican Republic :
+
+* Province [ref](https://es.wikipedia.org/wiki/C%C3%A1mara_de_Diputados_de_la_Rep%C3%BAblica_Dominicana) as province.
+* National District [ref](https://en.wikipedia.org/wiki/Distrito_Nacional) as state.
```

Comment:
Is there a particular reason to use `state` for the national district? If not, consider `district`

Reply:
Done.